src/constants/StyleConstants.ts

```diff
@@ -1 +1,10 @@
+const basicGrey = "rgb(236,236,236)";
+const trueWhite = "white";
```

Comment:
Can we be consistent in the type of color value format we're going to use? I'm actually a very big fan of hsl format (for [reasons](https://cloudfour.com/thinks/hsl-a-color-format-for-humans/)), but @tohuynh, @JacksonMaxfield, or @BrianL3 do you have any input/opinions?

Reply:
I don't have too big an opinion on this.... I had literally never heard of HSL had to look it up soooooo going to veto that one 😂 

RGB or RGBA seems good to me.

Reply:
Lol, I had never heard of HSL either up until a few months ago, but it's actually high-key an incredible color format with some really nice developer and design system benefits once you get used to it. I've seen it increasingly become used and loved in the frontend community in the last few months. 

You should definitely skim the last bit of the article I linked to before vetoing 😉.

Reply:
Gah. Alright I read through the article... It makes sense. I see the value in it. I think coming from an imaging background I am just used to RGB / RGBA. But here is where I will say: "I don't typically code on the frontend so my opinion shouldn't matter nearly as much as anyone else who works on this repo."

Idk. I get it. If you feel strongly about it then feel free to change it / wait for others opinions.

Reply:
HSL format doesn't seem any more human-readable to me than RGB.  The HSL "mathematically modify your colors" seems useful?  I have no strong opinions about the matter - with the notable exception of thinking that RGBA is bad.  

What I should be doing is actually importing all colors from the CDP Design project.

Reply:
Okay cool yeah I wanted to see what other opinions were, but it sounds like RGB is the way then!

I actually agree that HSL isn't intuitively human readable either. But it's more so that ability to mathematically manipulate sets of color that I think is really powerful.

However, it really isn't that hard to still use RGB, so I'm happy to go that route 🙂.

Reply:
Also yeah importing from CDP design would be great too.